cityofaustin/atd-moped
# remove completion date from project list view

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/19435

## Testing
**URL to test:** 

local

**Steps to test:**
- got to the `project_list_view` table and confirm the `completion_end_date` column is no longer present
- go to the project list view and confirm the 'Completion date' column is no longer present
- click the column list and confirm the field is no longer present
- confirm the csv export works and does not include the column
- go to the advanced filters and confirm the field is not present
- run the down migration and confirm the `completion_end_date` reappears in the `project_list_view` table

---
#### Ship list
